.github/actions/parse-version/action.yml

```diff
@@ -1,14 +1,16 @@
-name: "Parse versions"
-description: "Parse the major and minor version of Headless and Atomic"
+name: 'Parse versions'
+description: 'Parse the major and minor version of Headless and Atomic'
 outputs:
   headless-major:
-    description: "The major version of Headless (e.g., 2)"
+    description: 'The major version of Headless (e.g., 2)'
   headless-minor:
-    description: "The minor version of Headless (e.g., 2.12)"
+    description: 'The minor version of Headless (e.g., 2.12)'
   atomic-major:
-    description: "The major version of Atomic (e.g., 2)"
+    description: 'The major version of Atomic (e.g., 2)'
   atomic-minor:
-    description: "The minor version of Atomic (e.g., 2.12)"
+    description: 'The minor version of Atomic (e.g., 2.12)'
 runs:
+  # TODO(KIT-2443): Update to node LTS, currently yaml schema allows only: 12, 16
+  #   https://json.schemastore.org/github-action.json: "enum": ["node12", "node16"]
```

Comment:
@mrrajamanickam-coveo : https://github.com/orgs/community/discussions/53217#discussioncomment-6758913
We should probably jump straight to 20 when available, as 16 is croaking in 3 weeks.

Reply:
👍 Wish they also had a `LTS` tag to use